dapp/src/components/LedgerDerivationContent.js

```diff
@@ -141,6 +357,46 @@ const LedgerDerivationContent = ({}) => {
           min-height: 50px;
           width: 100%;
         }
+
+        .button-arrow {
+          width: 70px !important;
+          height: 35px !important;
+          padding: 0 !important;
+          margin: 10px 0 0 0 !important;
+        }
+
+        .arrow-icon {
+          width: 25px;
+          height: 25px;
+        }
+
+        .waiting-icon {
+          width: 25px;
+          height: 25px;
+        }
+
+        .rotating {
+          -webkit-animation: spin 2s linear infinite;
+          -moz-animation: spin 2s linear infinite;
+          animation: spin 2s linear infinite;
+        }
+
+        @-moz-keyframes spin {
```

Comment:
Are vendor prefix needed? It should be on by default as per the docs for `styled-jsx` 
https://github.com/vercel/styled-jsx#vendorprefixes